### ISSUE TITLE
Investigate slow installation (merging file trees) on windows

### ISSUE THREAD

**Issue by hasufell** [haskell/ghcup-hs#1164]
![Screenshot_2024-11-25_14-21-15](https://github.com/user-attachments/assets/d9ab11d4-beda-4119-b065-a5e1a0b21efd)

**Comment by andreasabel** [comment]
Yes, this is really very slow, on my (old) Windows machine this step takes more than 15min.

**Comment by FranklinWilson** [comment]
Recently had the same experience with very slow 'merging file tree'
As [andreasabel](https://github.com/andreasabel) mentioned, turning off _real-time protection (_Windows Security_ -> _Virus and threat protection_ -> _manage settings_) made all the difference, completed in a matter of seconds.
